### PR TITLE
Docs for pooler connection option

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
- Connection pooling:
https://neon-next-git-dprice-doc-for-pooler-option-neondatabase.vercel.app/docs/connect/connection-pooling#enable-connection-pooling
- Prisma Migrate content:
https://neon-next-git-dprice-doc-for-pooler-option-neondatabase.vercel.app/docs/guides/prisma#prisma-migrate-with-pgbouncer
- Release note update:
https://neon-next-git-dprice-doc-for-pooler-option-neondatabase.vercel.app/docs/release-notes/2023-02-06

### CODE REVIEW

**Comment by danieltprice** [comment]
Questions: 
- Prisma recommends adding ?pgbouncer=true to connection strings when using Prisma Client with PGBouncer from serverless function: https://www.prisma.io/docs/guides/performance-and-optimization/connection-management/configure-pg-bouncer#add-pgbouncer-to-the-connection-url
Do we believe this would no longer be required for a Neon connection string that uses the -pooler option?
- Does the `-pooler` option only work in connection strings or can I add it to the hostname in an app that uses a PGHOST variable?

**Comment by ololobus** [comment]
> Does the -pooler option only work in connection strings or can I add it to the hostname in an app that uses a PGHOST variable?

Yes, you can

File: content/docs/connect/connection-pooling.md [hunk 1]
```diff
@@ -5,28 +5,51 @@ redirectFrom:
...
+1. Find the branch with endpoint you want to enable pooling for, click the kebab menu in the **Compute endpoints** table, and select **Edit**.
+1. Toggle **Pooler enabled** to the on position.
+1. Click **Save**.
+
+You can also enable connection pooling when [creating a compute endpoint](/docs/manage/endpoints#create-a-compute-endpoint).
+
+### Enable pooling for individual connections
+
+This method enables pooling for individual connections that specify a `-pooler` option. Connection requests that use the `-pooler` option are directed to a connection pooler port. Connections that do not use the `-pooler` option connect directly to the database. This method supports workflows that require both pooled and non-pooled connections to the same database.
+
```
Comment (ololobus) [inline content/docs/connect/connection-pooling.md:39]: I'd first define what `-pooler` option means. It isn't actually an option, it's a suffix for the endpoint id in the host subdomain. Not sure how to properly word it, but for now it pops out from nowhere

Also this

> option are directed to a connection pooler port

sounds like internal implementation detail. I think we'd better just state with simple words that without this suffix you get a direct connection to the Postgres, and with connection pooler is used.

**Review by ololobus — COMMENTED** [review]
Everything sounds technically correct to me, but left some structural / style comment

File: content/docs/connect/connection-pooling.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/connect/connection-pooling.md]: Thank you. I've revised the wording, drop mention of a connection pooler port, and changed "option" to "suffix"

File: content/docs/connect/connection-pooling.md [hunk 2]
```diff
@@ -5,28 +5,35 @@ redirectFrom:
...
+Each PostgreSQL connection creates a new process in the operating system, which consumes resources. For this reason, PostgreSQL limits the number of open connections. Neon permits 100 simultaneous PostgreSQL connections by default with a `max_connections=100` setting, which is the typical default for this parameter. In Neon, a small number of those connections are reserved for administrative purposes. A connection limit of 100 may not be sufficient for some applications. To increase the number of connections that Neon supports, you can enable connection pooling.
```
Comment (ololobus) [inline content/docs/connect/connection-pooling.md:8]: > In Neon, a small number of those connections are reserved for administrative purposes

Not only in Neon, it's typical for any Postgres

File: content/docs/connect/connection-pooling.md [hunk 3]
```diff
@@ -5,28 +5,35 @@ redirectFrom:
...
 
 ## Connection pooling
 
-Some applications open numerous connections, with most eventually becoming inactive. This behavior can often be attributed to database driver limitations or to running many instances of an application. With regular PostgreSQL, new connections are rejected when reaching the `max_connections` limit. To overcome this limitation, Neon supports connection pooling using [PgBouncer](https://www.pgbouncer.org/).
+Some applications open numerous connections, with most eventually becoming inactive. This behavior can often be attributed to database driver limitations, to running many instances of an application, or to applications with serverless functions. With regular PostgreSQL, new connections are rejected when reaching the `max_connections` limit. To overcome this limitation, Neon supports connection pooling using [PgBouncer](https://www.pgbouncer.org/), allowing Neon to support up to 1000 concurrent connections.
```
Comment (ololobus) [inline content/docs/connect/connection-pooling.md:12]: @kelvich bumped it to 10000 recently

**Review by ololobus — APPROVED** [review]
I put a couple of comments, but generally looks good to me

File: content/docs/connect/connection-pooling.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/connect/connection-pooling.md]: Dropped the reference to Neon.

File: content/docs/connect/connection-pooling.md (hunk 3 above)
Comment (danieltprice) [reply content/docs/connect/connection-pooling.md]: Changed to 10000.

**Review by ololobus — APPROVED** [review]

**Pull request merged**